src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -242,13 +242,71 @@ Construct the regular icosahedron, one out of two exceptional Platonic solids.
 """
 icosahedron() = polyhedron(Polymake.polytope.icosahedron());
 
-const _johnson_indexes_from_oscar = Set{Int}([9, 10, 13, 16, 17, 18, 20, 21, 22, 23, 24,
-                                              25, 30, 32, 33, 34, 35, 36, 38, 39, 40,
-                                              41, 42, 43, 44, 45, 46, 47, 48, 49, 50,
-                                              51, 52, 53, 54, 55, 56, 57, 58, 59, 60,
-                                              61, 64, 68, 69, 70, 71, 72, 73, 74, 75,
-                                              77, 78, 79, 82, 84, 85, 86, 87, 88, 89,
-                                              90, 92])
+const _johnson_indexes_from_oscar = Set{Int}([
+  9,
+  10,
+  13,
+  16,
+  17,
+  18,
+  20,
+  21,
+  22,
```

Comment:
Hmm, I'm not sure I like this change. If one scrolls down to the end of the list, one can't see the name `_johnson_indexes_from_oscar` anymore, at least on my screen. Apparently, one could leave it as it were and add comments for the formatter to ignore it (https://domluna.github.io/JuliaFormatter.jl/stable/#Turn-off/on-formatting) so that the tests pass, but I did not test this.

But you can also leave it like this; long lists of constants never look great I guess.

Reply:
I like the idea of putting *some few* noformat tags around things like this to keep files readable. That's why we even format them in the first place imo.